src/main/java/it/pagopa/interop/probing/eservice/operations/service/impl/EserviceServiceImpl.java

```diff
@@ -102,10 +105,8 @@ public void updateEserviceProbingState(UpdateEserviceProbingStateDto inputData)
 
     eServiceToUpdate.probingEnabled(inputData.isProbingEnabled());
     eserviceRepository.save(eServiceToUpdate);
-
-    log.info("EserviceProbingState of eservice " + eServiceToUpdate.eserviceId() + " with version "
-        + eServiceToUpdate.versionId() + " has been updated into "
-        + eServiceToUpdate.probingEnabled());
+    logger.logMessageEserviceProbingStateUpdated(eServiceToUpdate.eserviceId(),
```

Comment:
Maybe it is better to pass the whole object

Reply:
Done and pushed